.github/workflows/merge-node-exporter.yaml

```diff
@@ -15,6 +15,8 @@ jobs:
       restore-downstream: >-
          OWNERS
       restore-upstream: >-
```

Comment:
nit: could we please maintain the list sorted .. i.e. 
```
CHANGELOG.md
VERSION
collector
go.mod
go.sum
```

Reply:
@sthaha Fixed it. Thank you.